src/tests/support.py

```diff
@@ -0,0 +1,19 @@
+import os
```

Comment:
Do you need to sync the branch?

Reply:
I don't think that is necessary, since we merged the tests branch into this one (not into master).

Reply:
oh! I didn't realise that.

In future we probably want to merge into master unless there's a good reason.